src/cmd-build

```diff
@@ -452,18 +452,31 @@ else
                  "--label=io.openshift.build.versions=machine-os=${buildid}"
                 )
     fi
+    
+    last_build_manifest=()
+    #Use the last stable release if buildfetch used
```

Comment:
I'd still add spaces between the comment character and the start of the text here too.